packages/grafana-ui/.storybook/webpack.config.js

```diff
@@ -7,30 +7,39 @@ module.exports = ({ config, mode }) => {
     ...(config.module.rules || []),
     {
       test: /\.tsx?$/,
-      use: [
-        {
-          loader: require.resolve('ts-loader'),
-          options: {
-            // transpileOnly: true,
-            configFile: path.resolve(__dirname, 'tsconfig.json'),
-          },
-        },
-        {
-          loader: require.resolve('react-docgen-typescript-loader'),
-          options: {
-            tsconfigPath: path.resolve(__dirname, 'tsconfig.json'),
-            // https://github.com/styleguidist/react-docgen-typescript#parseroptions
-            // @ts-ignore
-            propFilter: prop => {
-              if (prop.parent) {
-                return !prop.parent.fileName.includes('node_modules/@types/react/');
-              }
+      use: isProductionBuild
```

Comment:
Good call, but this will result in the props table not being rendered in the dev mode's preview. I suggest maybe use env variables instead of the production assumption only (https://storybook.js.org/docs/configurations/env-vars/) and introduce another storybook script in the grafana-ui package.json that would allow opting in to render the docs tables too? like `yarn storybook:docs`?

Reply:
yes good idea.